data/scenarios/Tutorials/conditionals.yaml

```diff
@@ -25,28 +25,28 @@ objectives:
         it is simply a built-in function of type
       - |
         ```
-        if : bool -> {a} -> {a} -> a
+        if : ∀ a. bool -> {a} -> {a} -> a
```

Comment:
Is this syntax introduced before? If players did not know the `forall`/`∀` then this would be confusing.